services/batch-service.js

```diff
@@ -187,12 +187,13 @@ class BatchService {
      * @returns string
      */ 
     async getMetadataFile(submission, aBatch, fileName) {
+        const submissionName = submission.name.replace("/", "_");
         if(fileName){
             const file = aBatch?.files?.find(f=>f.fileName === fileName && f.status === FILE.UPLOAD_STATUSES.UPLOADED);
             if(!file){
                 throw new Error(ERROR.FILE_NOT_EXIST);
             }
-            const output_filename =  `${submission.name}_metadata_batch${aBatch.displayID}${fileName}`;
+            const output_filename = `${submissionName}_metadata_batch${aBatch.displayID}${fileName}`;
```

Comment:
Could you use outputFilename and update all variables to follow camelCase naming?